Revit_Engine/Create/Requests/FilterRequest.cs

```diff
@@ -0,0 +1,46 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Data.Requests;
+using BH.oM.Reflection.Attributes;
+using System;
+using System.ComponentModel;
+
+namespace BH.Engine.Adapters.Revit
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Creates an IRequest that filters Revit elements based on their correspondent BHoM Type. Wrapper for BH.oM.Data.Requests.FilterRequest.")]
+        [Input("bHoMType", "BHoM type, which correspondent Revit types are meant to be filtered.")]
+        [Output("F", "FilterRequest to be used to filter Revit elements based on their correspondent BHoM Type.")]
+        public static FilterRequest FilterByBHoMType(Type bHoMType)
+        {
+            return new FilterRequest { Type = bHoMType };
+        }
```

Comment:
I can't really see the use for this one: the base FilterRequest create already does the same, you can Create it only with the Type (tag defaults to ""). Why do you need this?
Additionally, if you call it `FilterByBHoMType`, I'd have a check on the Type to make sure that it is actually a BHoM Type (reflection namespace startswith "BH")

Reply:
I will let myself cite my own words:
> For the UX purposes, FilterByBHoMType wrapper for FilterRequest as well as FilterBySelection for SelectionRequest have been added.

In general, as explained in the PR description, I wanted to create a clear scheme for the end user, i.e. all Revit requests starting with _Filter_ prefix. `FilterRequest` means nothing to the end user, plus it will not appear in the Revit namespace == total confusion of the end user.

`FilterByBHoMType` indicates that it can go in combination with BHoM `CreateType` component.